macro/readEMCHits.C

```diff
@@ -0,0 +1,49 @@
+#if !defined(__CLING__) || defined(__ROOTCLING__)
+
+#include <TFile.h>
+#include <TTree.h>
+#include <TStopwatch.h>
+#include <memory>
+#include "FairLogger.h"
+#include "DataFormatsITSMFT/ROFRecord.h"
+#include "ITSMFTBase/Digit.h"
+#include "SimulationDataFormat/RunContext.h"
+#include "SimulationDataFormat/MCTruthContainer.h"
+#include "SimulationDataFormat/MCCompLabel.h"
+
+#endif
+/// Example of accessing the digits of MCEvent digitized with continous readout
+
+void readEMCHits(std::string path = "./",
+                 std::string mcfileName = "o2sim.root")
```

Comment:
Dear @matthiasrichter,

This is a test macro, copied and converted from readITSDigits.C. Despite some parts which need to be fixed as you discussed below I nevertheless would like to have the file committed somewhere as we probably will still use it for a while in the development phases. If it is not appropriate to have this macro in the repository (which I understand as the repo can become trashed) I wonder how then similar macros could end up there.

Cheers

Markus

Reply:
Hi Markus, actually, I was only wondering because the PR title or description did not say anything about the macro, I noticed  later that the commit message is mentioning it. I would suggest to cleanup it a little bit before merging.